hoomd/md/pytest/test_hdf5.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2009-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import numpy as np
+import pytest
+
+from hoomd.conftest import operation_pickling_check
+import hoomd
+import hoomd.write
+
+try:
+    import h5py
+except ImportError:
+    pytestmark = pytest.skip("h5py required to test this feature.")
+
+
+def lj_integrator():
+    integrator = hoomd.md.Integrator(dt=0.005)
+    lj = hoomd.md.pair.LJ(nlist=hoomd.md.nlist.Cell(buffer=0.4),
+                          default_r_cut=2.5)
+    lj.params.default = {'sigma': 1, 'epsilon': 1}
+    integrator.forces.append(lj)
+    langevin = hoomd.md.methods.Langevin(hoomd.filter.All(), kT=1)
+    integrator.methods.append(langevin)
+
+    return integrator
+
+
+@pytest.fixture(scope='function')
+def create_md_sim(simulation_factory, device, two_particle_snapshot_factory):
+    sim = simulation_factory(two_particle_snapshot_factory())
+    sim.operations.integrator = lj_integrator()
+
+    return sim
+
+
+@pytest.fixture
+def logger():
+    logger = hoomd.logging.Logger(categories=['scalar'])
+    return logger
```

Comment:
Does a very simple one-liner like this warrant a fixture?

Reply:
Not really, just a stylistic preference, but I will add the line to the tests.